plugin/db/mysql/restore.go

```diff
@@ -617,15 +617,14 @@ func (driver *Driver) downloadBinlogFile(ctx context.Context, binlogFileToDownlo
 	if isLast {
 		// Case 1: It's the last binlog file we need (contains the targetTs).
 		// If it's the last (incomplete) binlog file on the MySQL server, it will grow as new writes hit the database server.
-		// We just need to check that the downloaded file size >= queried size, so it contains the targetTs event.
+		// Chances are that the latest binlog file size < size queried on the MySQL server, maybe because of some file sync issue.
+		// We can safely ignore this, because a later download will rewrite this binlog file, and it only causes the binlog replay to failed when the targetTs is just in the last not synced range.
```

Comment:
```suggestion
		// We can safely ignore this, because a later download will rewrite this binlog file, and it only causes the binlog replay to fail when the targetTs is just in the last not synced range.
```